app/src/components/About.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div
     :class="[showAbout ? 'flex' : 'hidden']"
-    class="h-screen fixed w-full"
-    style="z-index: 10000; background-color: #ffffffdd"
+    class="h-screen fixed w-full z-50 bg-white bg-opacity-80"
+    @click="emitEvent('toggleAbout')"
   >
     <div class="px-4 md:px-12 m-auto bg-white">
-      <div class="border-grey-500 border-2 px-4 md:px-12 relative">
+      <div class="border-grey-500 border-2 px-4 md:px-12 relative" @click.stop>
         <div class="absolute top-6 right-6">
           <XIcon @click="$emit('toggleAbout')" class="icon icon-primary cursor-pointer" />
```

Comment:
Should change `$emit` to the `emitEvent` method

Reply:
Missed that one. Fixed!